internal/handlers/ping.go

```diff
@@ -0,0 +1,14 @@
+package handlers
+
+import (
+	"net/http"
+)
+
+// PingDB проверяет подключение к базе данных
+func (h *Handler) PingDB(w http.ResponseWriter, r *http.Request) {
+	if err := h.db.Ping(); err != nil {
+		http.Error(w, "Database connection error", http.StatusInternalServerError)
+		return
+	}
+	w.WriteHeader(http.StatusOK)
```

Comment:
Если сделать вот так можно не использовать `MockDB`.
```suggestion
	if h.db != nil {
		if err := h.db.Ping(); err != nil {
			http.Error(w, "Database connection error", http.StatusInternalServerError)
			return
		}
	}
	w.WriteHeader(http.StatusOK)
```